docs/pbs/advanced-pbs-topics.md

```diff
@@ -0,0 +1,460 @@
+
+|  Clause | Effect |
+|---------|--------|
+| `after:<arg_list>`      | This job may be scheduled for execution at any point after all jobs in `<arg_list>` have *started* execution. |
+| `afterok:<arg_list>`    | This job may be scheduled for execution only after all jobs in `<arg_list>` have terminated with no errors. |
+| `afternotok:<arg_list>` | This job may be scheduled for execution only after all jobs in `<arg_list>` have terminated with errors. |
+| `afterany:<arg_list>`   | This job may be scheduled for execution after all jobs in `<arg_list>` have finished execution, with any exit status (with or without errors.) This job will not run if a job in the `<arg_list>` was deleted without ever having been run. |
+| `before:<arg_list>`      | Jobs in `<arg_list>` may begin execution once this job has begun execution.|
+| `beforeok:<arg_list>`    | Jobs in `<arg_list>` may begin execution once this job terminates without errors. |
+| `beforenotok:<arg_list>` | If this job terminates execution with errors, jobs in `<arg_list>` may begin. |
+| `beforeany:<arg_list>`   | Jobs in `<arg_list>` may begin execution once this job terminates execution, with or without errors.|
+
+## Peer Scheduling scheduling between systems
+
+!!! info "Peer Scheduling scheduling between HPC Systems"
+    Derecho and Casper use the PBS scheduler, and each system has its own
+    dedicated PBS server to manage job submission and execution. These
+    "peer" servers can share data between each other, and *jobs can be
+    submitted from one system to another*. It is also possible to create
+    dependencies between jobs on each server, enabling simulation-analysis
+    workflows that target the appropriate system for each task.
+
+### Submitting a job to a peer system
+
+To submit a job to a queue on a peer server, you need to append the name
+of the server to the queue directive in your job script. The names of
+the PBS servers are:
+
+| **System** | **PBS server name**            |
+|------------|--------------------------------|
+| Casper     | `casper-pbs`                    |
+| Derecho    | `desched1` |
+
+!!! warning
+    Do not use the qsub option `-W block=true` when peer scheduling. PBS will
+    not be able to tell when the job is finished and the qsub call will hang
+    indefinitely.
+
+#### Examples
+=== "Casper to Derecho"
+    You want to submit to the Derecho "main" queue from a
+    Casper login node or compute node. Append the Derecho server name as
+    follows in your job script when specifying the queue:
+    ```pre
+    #PBS -q main@desched1
+    ```
+
+=== "Derecho to Casper"
+    You want to submit a job to Casper from Derecho. Append
+    the Casper server name as follows in your job script when specifying the
+    destination:
+    ```pre
+    #PBS -q casper@casper-pbs
+    ```
+
+The server-specific queue names will be understood by both PBS servers,
+so if you will want to submit the same script at times from Casper,
```

Comment:
I'd suggest "from the peer server" instead of "from Casper", since we have examples in both directions above.